demos/layout-grid.html

```diff
@@ -359,49 +424,83 @@ <h2 class="demo-grid-legend">Fixed column width layout grid</h2>
     <script>
       (function(global) {
         'use strict';
-        var marginSelectDesktop = document.querySelector('#desktop-margin');
-        marginSelectDesktop.addEventListener('change', function() {
-          document.documentElement.style.setProperty('--mdc-layout-grid-margin-desktop', marginSelectDesktop.value);
+        var fluidContainerSection = document.getElementById('layout-grid-in-fluid-container');
+        var fixedWidthSection = document.getElementById('fixed-column-width-layout-grid');
+
+        var marginSelectDesktopFluid = fluidContainerSection.querySelector('.desktop-margin');
+        marginSelectDesktopFluid.addEventListener('change', function() {
+          console.log(marginSelectDesktopFluid, marginSelectDesktopFluid.value);
```

Comment:
I'm guessing this `console.log` was unintentionally left in?

Reply:
True! Thanks for digging it out!